src/views/RemoveLiquidity.vue

```diff
@@ -204,10 +216,20 @@ export default class RemoveLiquidity extends Mixins(TransactionMixin, LoadingMix
         secondAddress: this.secondTokenAddress
       })
     })
+    this.sliderDragButton = this.$el.querySelector('.slider-container .el-slider__button')
+    this.sliderInput = this.$el.querySelector('.s-input--remove-part .el-input__inner')
+    if (this.sliderDragButton) {
+      this.sliderDragButton.addEventListener('mousedown', this.focusSliderInput)
+    }
+  }
+
+  destroyed () {
+    if (this.sliderDragButton) {
+      this.$el.removeEventListener('mousedown', this.focusSliderInput)
```

Comment:
You've added an event listener to `this.sliderDragButton`, so it should be removed from the same element:
`this.sliderDragButton.removeEventListener('mousedown', this.focusSliderInput)`

Reply:
Such a good point! thank you so much!